okpt/io/config/validator.py

```diff
@@ -0,0 +1,109 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+"""Provides validation for various config files.
+
+Functions:
+    validate(): Validate the tool configuration.
+
+Exceptions:
+    ConfigurationError: An error in the tool configuration.
+
+"""
+
+import os
+from io import TextIOWrapper
+from typing import Any, Dict
+
+import cerberus
+
+from okpt.io import reader
```

Comment:
what is okpt?

Reply:
Opensearch k-NN Testing Tool. I'm not sure if it's the best module name.